src/components/SettingsPage/ListPopulationComponent.js

```diff
@@ -113,19 +121,20 @@ export default function ListPopulationComponent() {
           deleteCategory={(category) => deleteCategory(category)}
         />
       </Grid>
-
+      }
       {/*for creating area list*/}
-      <Grid item xs={12} sm={5}>
+      {slug === 'area' &&
+      <Grid item xs={12} >
         <CustomInput
-          labelId="Area name"
+          labelId="areaName"
```

Comment:
Should it be title?

Reply:
used the exiting nameing and added to en.json, will update